app/helpers/SessionAnswersHelper.scala

```diff
@@ -148,6 +148,9 @@ class SessionAnswersHelper @Inject() (uploadJourneyRepository: UploadJourneyRepo
           (statementOfLatenessForLPPOrLSP,
             request.session.get(SessionKeys.whyReturnSubmittedLate).get,
             controllers.routes.OtherReasonController.onPageLoadForWhyReturnSubmittedLate(CheckMode).url),
+          (messages("checkYourAnswers.other.uploadEvidenceQuestion"),
```

Comment:
Sorry one more thing, I think you are missing the row for obligation appeals as well:
![image](https://user-images.githubusercontent.com/53828896/155501230-27f17e6a-9586-4afc-b9ab-579a3e7abab1.png)
Could you add it in and some tests please?

Reply:
Added row
<img width="822" alt="Screenshot 2022-02-24 at 10 55 45" src="https://user-images.githubusercontent.com/86231805/155511358-7132553c-385c-4fb3-8e36-b4609aed7185.png">